Airports/EGBE/Positions_Mentor.txt

```diff
@@ -1,2 +1 @@
-EGBE_APP:Coventry Radar:199.998:BEAM:E:EGBE:APP:-:-:0260:0267:N052.22.11.000:W001.28.47.000::
-EGBE_TWR:Coventry Tower:199.998:BETM:T:EGBE:TWR:-:-:0260:0267:N052.22.11.000:W001.28.47.000::
\ No newline at end of file
+EGBE_APP:Coventry Radio:199.998:BERM:E:EGBE:TWR:-:-:0260:0267:N052.22.11.000:W001.28.47.000::
```

Comment:
I don't think we have any radio mentor positions in the file elsewhere - I suggest we just remove this file.